src/services/EdFi.DataManagementService.Api.Tests.Unit/Core/Middleware/ValidateDocumentMiddlewareTests.cs

```diff
@@ -146,21 +147,49 @@ public async Task Setup()
         }
 
         [Test]
-        public void It_has_a_response()
+        public void It_should_not_have_response()
         {
-            _context?.FrontendResponse.Should().NotBe(No.FrontendResponse);
+            _context?.FrontendResponse.Should().Be(No.FrontendResponse);
         }
 
         [Test]
-        public void It_returns_status_400()
+        public void It_returns_status_503()
         {
-            _context?.FrontendResponse.StatusCode.Should().Be(400);
+            _context?.FrontendResponse.StatusCode.Should().Be(503);
+        }
+    }
+
+    [TestFixture]
+    public class Given_a_request_with_not_existing_nested_property : ValidateDocumentMiddlewareTests
+    {
+        private PipelineContext _context = No.PipelineContext();
+
+        [SetUp]
+        public async Task Setup()
+        {
+            string jsonData =
+                """{"schoolId": 989, "gradeLevels":{"gradeLevelDescriptor": "grade1", "gradeLevelOverPost": "overPostedValue"},"nameOfInstitution":"school12", "propertyOverPost": "overPostedValue"}""";
+
+            var frontEndRequest = new FrontendRequest(
+                RequestMethod.POST,
+                "ed-fi/schools",
+                Body: JsonNode.Parse(jsonData),
+                new TraceId("traceId")
+            );
+            _context = Context(frontEndRequest);
+            await Middleware().Execute(_context, Next());
+        }
+
+        [Test]
+        public void It_should_not_have_response()
+        {
+            _context?.FrontendResponse.Should().Be(No.FrontendResponse);
         }
 
         [Test]
-        public void It_returns_message_body_with_overpost_validation_error()
+        public void It_returns_status_503()
```

Comment:
You can delete this. On success this middleware doesn't return a response, which line 186 tests. The 503 is meaningless.

Separately, rename test to clarify overposting is what's being tested.